modules/nf-core/bamtofastq10x/main.nf

```diff
@@ -0,0 +1,54 @@
+process BAMTOFASTQ10X {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/10x_bamtofastq:1.4.1--hdbdd923_2':
+        'biocontainers/10x_bamtofastq:1.4.1--hdbdd923_2' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("${meta.id}/**/*.fastq.gz"), emit: fastq
+    path "versions.yml"                              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    bamtofastq \\
+        $args \\
+        $bam \\
+        $prefix
```

Comment:
Changed the output path from `${prefix}.fastq.gz`.

bamtofastq generates a directory containing two folders: one for GEX and one for CMO `.fastq` files.
The two folders are prefixed with the `.bam` prefix.
All files are automatically prefixed with `bamtofastq`.